### DISCUSSION TITLE
ACE paper is available for download!

### DISCUSSION THREAD

**Discussion by daveshap** [daveshap/ACE_Framework#9] [Announcements]
https://github.com/daveshap/ACE_Framework/blob/main/publications/Conceptual%20Framework%20for%20Autonomous%20Cognitive%20Entities%20(ACE).pdf

**Comment by Glasperlin** [comment]
Dear Dave: I am thrilled with what you have accomplished and would like to participate. I'm attaching a PPT so that you see what I can bring to the project. More about me at www.mtkluminous.com

[Regenerative Civilization Public 230804.pptx](https://github.com/daveshap/ACE_Framework/files/12805743/Regenerative.Civilization.Public.230804.pptx)

**Reply by agusavior** [reply]
May I publish your paper in my own website?
I will redirect to [this link](https://github.com/daveshap/ACE_Framework/blob/main/publications/Conceptual%20Framework%20for%20Autonomous%20Cognitive%20Entities%20(ACE).pdf).

**Reply by Glasperlin** [reply]
Best not, its a draft and I have not assessed the copyright status of the photos. It's intended as a way to raise interest in creating the AGI it describes. If you wish pass it around for that purpose, I do not think that would be a problem.

Regards,

Mike
